### ISSUE TITLE
replatform: redirect from team to author is not implemented

### ISSUE THREAD

**Issue by cantsin** [18F/18f.gsa.gov#4043]
Example:

https://18f.gsa.gov/team/tadhg/ goes to https://18f.gsa.gov/author/tadhg/

We have no such redirect for the replatformed effort; I think we missed this directory-level redirect. 

Note: other per-page redirections work. `/project/eregulations/` goes to `/our-work/eregulations/`, for instance.

**Comment by beechnut** [comment]
Okay so — the redirects are specified at the individual author level in the file inside `_data/`. I don't have the whole solution in mind, but my first step is to look into adding redirect metadata to the 11ty data file that turns `_data/authors` into pages.

**Comment by beechnut** [comment]
There's a tricky indirection here: the redirects are all handled by `content/redirect.html`. 

I tried adding

```yaml
# content/pages/author.liquid
redirect_from: "{{ author.redirect_from }}"
```

but this creates a folder `_site/{{ author.redirect_from }}`, which is not what we want.


And this seems to do nothing at all:
```yaml
# content/pages/author.liquid
eleventyComputed:
  redirect_from: "{{ author.redirect_from }}"
```

There's a `_site/team/18F/index.html` but it could be coming from somewhere other than the author data.

**Comment by beechnut** [comment]
Okay, a challenge question: does this matter enough to be part of the migration? I think we've been redirecting from `/team` urls for years, to the point that I'd be surprised if anyone is still pinging those.

Do we have analytics with which to test the hypothesis? Can we see if anyone's touched `/team/*` in the last 3 years?

**Comment by cantsin** [comment]
So, the thing is that GA does not track redirects, or at least it won't tell us where the redirect came from (afaics). So I do not know if we have any real way of knowing.

I looked at the top 500 visited paths for the last year (the limit is 500) and `author` appears quite a few times. 

I tried being more explicit:

![20240807_06h25m10s_grim](https://github.com/user-attachments/assets/89fd5053-e75a-4da6-88da-30525e98c81f)
gives us no matches whatsoever.

I'm comfortable punting on this one until we find concrete evidence otherwise.
